examples/app.py

```diff
@@ -29,12 +29,20 @@
 
 .. code-block:: console
 
-    $ # install the sample application
-    $ cd examples/samplepkg
-    $ pip install -e .
-    $ # run the example application
-    $ cd ..
+    $ pip install -e .[all]
+    $ ./app-setup.sh
```

Comment:
This would not "propagate" instructions into documentation, but I'll reply you on this one in the other PR where we started talking about using shell scripts.